### PR TITLE
Header

### PR DESCRIPTION
**Pull request by artur1602**

### CODE REVIEW

File: src/partials/header.html [hunk 1]
```diff
@@ -1 +1,180 @@
-<header>header</header>
+
+    <!--HEADER-->
+    <header class="header" id="home">
+      <div class="header-container container" id="header-top">
+        <!-- Navigation -->
+        <a href="" class="header-logo"
```
Comment (asaulyak) [inline src/partials/header.html:6]: add a link to index.html

File: src/partials/header.html [hunk 2]
```diff
@@ -1 +1,180 @@
...
+          <li class="header-socials-item">
+            <a
+              class="header-socials-link link"
+              href="https://facebook.com"
+              target="_blank"
+              rel="noopener noreferrer"
+              aria-label="Facebook link"
+            >
+              <svg class="header-socials-icon" width="28" height="28">
+                <use href="./images/icons.svg#icon-facebook"></use>
```
Comment (asaulyak) [inline src/partials/header.html:42]: images live in `img` folder

File: src/partials/header.html [hunk 3]
```diff
@@ -1 +1,180 @@
...
+          type="button"
+          aria-expanded="false"
+          aria-controls="mobile-menu"
+          data-menu-open-btn
+        >
+          <svg
+            class="menu-open-icon"
+            width="32"
+            height="23"
+            stroke="var(--color-black)"
```
Comment (asaulyak) [inline src/partials/header.html:86]: move stroke to css

File: src/partials/header.html [hunk 4]
```diff
@@ -1 +1,180 @@
...
+            aria-controls="mobile-menu"
+            aria-expanded="false"
+            role="img"
+          >
+            <use
+              class="menu-open-use"
+              xlink:href="./images/icons.svg#icon-Nav"
```
Comment (asaulyak) [inline src/partials/header.html:93]: remove xlink:
add class to `svg` tag, not  `use`

File: src/partials/header.html [hunk 5]
```diff
@@ -1 +1,180 @@
...
+            data-menu-close-btn
+          >
+            <svg class="menu-close-icon" width="32" height="32">
+              <use href="./images/icons.svg#icon-Nav"></use>
+            </svg>
+          </button>
+
+          <!-- Mobile Menu Navigation  -->
+
+          <nav class="menu-nav">
```
Comment (asaulyak) [inline src/partials/header.html:113]: mobile navigation should not be surrounded with `nav` tag
use `div` instead

File: src/img/header/icons.svg
Comment (asaulyak) [inline src/img/header/icons.svg:1]: remove

File: src/img/header/instagram.svg
Comment (asaulyak) [inline src/img/header/instagram.svg:1]: remove

File: src/img/header/x.svg
Comment (asaulyak) [inline src/img/header/x.svg:1]: remove

File: src/img/header/youtube.svg
Comment (asaulyak) [inline src/img/header/youtube.svg:1]: remove

File: src/partials/header.html [hunk 6]
```diff
@@ -1 +1,174 @@
-<header>header</header>
+<!--HEADER-->
+<header class="header" id="home">
```
Comment (asaulyak) [inline src/partials/header.html:2]: remove either id="home" or id="header-top"

File: src/partials/header.html [hunk 7]
```diff
@@ -1 +1,174 @@
...
+  <div class="header-container container" id="header-top">
+    <!-- Navigation -->
+    <a href="../index.html" class="header-logo"
```
Comment (asaulyak) [inline src/partials/header.html:5]: href="./index.html"

File: src/partials/header.html [hunk 8]
```diff
@@ -1 +1,174 @@
...
+      <li class="header-socials-item">
+        <a
+          class="header-socials-link link"
+          href="https://facebook.com"
+          target="_blank"
+          rel="noopener noreferrer"
+          aria-label="Facebook link"
+        >
+          <svg class="header-socials-icon" width="28" height="28">
+            <use href="../img/header/icons.svg#icon-facebook"></use>
```
Comment (asaulyak) [inline src/partials/header.html:39]: use ./img/icons.svg

File: src/partials/header.html [hunk 9]
```diff
@@ -1 +1,174 @@
...
+      <li class="header-socials-item">
+        <a
+          class="header-socials-link link"
+          href="https://instagram.com"
+          target="_blank"
+          rel="noopener noreferrer"
+          aria-label="Instagram link"
+        >
+          <svg class="header-socials-icon" width="28" height="28">
+            <use href="../img/header/icons.svg#icon-instagram"></use>
```
Comment (asaulyak) [inline src/partials/header.html:52]: use ./img/icons.svg

File: src/partials/header.html [hunk 10]
```diff
@@ -1 +1,174 @@
...
+      <li class="header-socials-item">
+        <a
+          class="header-socials-link link"
+          href="https://youtube.com"
+          target="_blank"
+          rel="noopener noreferrer"
+          aria-label="youtube link"
+        >
+          <svg class="header-socials-icon" width="24" height="24">
+            <use href="../img/header/icons.svg#icon-youtube"></use>
```
Comment (asaulyak) [inline src/partials/header.html:65]: use ./img/icons.svg

File: src/partials/header.html (hunk 1 above)
Comment (artur1602) [reply src/partials/header.html]: +

File: src/partials/header.html (hunk 3 above)
Comment (artur1602) [reply src/partials/header.html]: +

File: src/partials/header.html (hunk 4 above)
Comment (artur1602) [reply src/partials/header.html]: +

File: src/partials/header.html (hunk 5 above)
Comment (artur1602) [reply src/partials/header.html]: +

File: src/partials/header.html (hunk 3 above)
Comment (artur1602) [reply src/partials/header.html]: +

File: src/css/variables.css [hunk 11]
```diff
@@ -10,4 +10,6 @@
   --soft-white: rgba(255, 255, 255, 0.5);
   --grey-bg-image: rgb(128, 127, 127);
   --animation-fast: 250ms cubic-bezier(0.4, 0, 0.2, 1);
+	--transition-dur-and-func: 0.75s cubic-bezier(0.4, 0, 0.2, 1);
```
[the remark this answers is not in the document]
Comment (artur1602) [reply src/css/variables.css]: +

File: src/js/form.js [hunk 12]
```diff
@@ -0,0 +1,17 @@
+const useElement = document.querySelector('.menu-open__use');
```
[the remark this answers is not in the document]
Comment (artur1602) [reply src/js/form.js]: +

File: src/img/header/Nav.svg
Comment (asaulyak) [inline src/img/header/Nav.svg:1]: remove

File: src/img/header/facebook.svg
Comment (asaulyak) [inline src/img/header/facebook.svg:1]: remove

File: src/css/header.css [hunk 13]
```diff
@@ -0,0 +1,504 @@
...
+  }
+}
+
+@media screen and (min-width: 1280px) {
+  .container {
+    width: 1280px;
+  }
+}
+
+.visually-hidden {
```
Comment (asaulyak) [inline src/css/header.css:35]: remove styles from lines 35-78

File: src/css/header.css [hunk 14]
```diff
@@ -0,0 +1,504 @@
...
+
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  min-width: 320px;
+  margin: 0 auto;
+}
+
+@media screen and (min-width: 375px) {
```
Comment (asaulyak) [inline src/css/header.css:88]: remove rule

File: src/css/header.css [hunk 15]
```diff
@@ -0,0 +1,504 @@
...
+  .header-container {
+    width: 375px;
+  }
+}
+
+@media screen and (min-width: 768px) {
```
Comment (asaulyak) [inline src/css/header.css:94]: remove rule

File: src/css/header.css [hunk 16]
```diff
@@ -0,0 +1,504 @@
...
+  .header-container {
+    width: 768px;
+  }
+}
+
+@media screen and (min-width: 1280px) {
```
Comment (asaulyak) [inline src/css/header.css:100]: remove rule

File: src/css/header.css [hunk 17]
```diff
@@ -0,0 +1,504 @@
...
+}
+
+.header-socials-link {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 10px;
+  border: 1px solid rgba(18, 20, 23, 0.1);
```
Comment (asaulyak) [inline src/css/header.css:300]: use color var

File: src/css/header.css [hunk 18]
```diff
@@ -0,0 +1,504 @@
...
+}
+
+.header-socials-link:hover,
+.header-socials-link:focus {
+  border: 1px solid rgba(77, 194, 116, 0.1);
```
Comment (asaulyak) [inline src/css/header.css:305]: use color var

File: src/css/header.css [hunk 19]
```diff
@@ -0,0 +1,504 @@
...
+  transition: border var(--animation-fast);
+}
+
+.header-socials-link:hover .header-socials-icon,
+.header-socials-link:focus .header-socials-icon {
+  fill: #4dc274;
```
Comment (asaulyak) [inline src/css/header.css:311]: use color var

File: src/partials/header.html [hunk 20]
```diff
@@ -1 +1,174 @@
...
+          <a class="header-nav-link-two" href="#our-lessons">Our lessons</a>
+        </li>
+        <li class="header-nav-item">
+          <a class="header-nav-link-three" href="#teachers">Teachers</a>
+        </li>
+        <li class="header-nav-item">
+          <a class="header-nav-link-four" href="#reviews">Reviews</a>
+        </li>
+        <li class="header-nav-item">
+          <a class="header-nav-link-five" href="#leave-section">Contact Us</a>
```
Comment (asaulyak) [inline src/partials/header.html:25]: `<a class="header-nav-link-five" href="#leave-application">Contact Us</a>`

File: src/css/header.css (hunk 17 above)
Comment (AndriyBabiy) [reply src/css/header.css:300]: had to create a new variable color "--ultra-soft-black"

**Review by asaulyak — APPROVED** [review]

**Pull request merged**